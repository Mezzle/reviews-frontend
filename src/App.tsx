import * as React from 'react';
import Home from './sections/Home';

const App: React.SFC<{}> = props => <Home />;

export default App;
