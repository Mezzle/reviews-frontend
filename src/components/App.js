/*
 * Copyright (c) 2019 Martin Meredith <martin@sourceguru.net>
 */

import React from 'react';
import { ConnectedRouter } from 'connected-react-router';
import { Route } from 'react-router-dom';
import routes from '../routes';

const App = ({ history }) => (
  <ConnectedRouter history={history}>
    <React.Suspense fallback={<div>Loading...</div>}>
      {routes.map(route => (
        <Route {...route} key={route.path} />
      ))}
    </React.Suspense>
  </ConnectedRouter>
);

export default App;
