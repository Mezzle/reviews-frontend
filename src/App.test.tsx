import { shallow } from 'enzyme';
import * as React from 'react';
import App from './App';

describe('Main app', () => {
  it('renders consistently', () => {
    const component = shallow(<App />);

    expect(component).toMatchSnapshot();
  });
});
