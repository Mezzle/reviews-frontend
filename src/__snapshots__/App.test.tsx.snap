// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main app renders consistently 1`] = `<Connect(HomeContainer) />`;
